src/main/scala/org/constellation/util/Signature.scala

```diff
@@ -11,15 +11,17 @@ import org.constellation.schema.Id
 
 trait Signable {
 
+  protected def toEncode: AnyRef = this
+
```

Comment:
Cool, remember to implement the same in wallet.

Reply:
:+1: